Automattic/wp-calypso
# Paid stats: Update Upsell to match design

Related to https://github.com/Automattic/red-team/issues/304

## Proposed Changes

* Add Lock Icon
* Updated copy
* Adjusts width / spacing to more closely match line breaks
* Remove NavigationPeriod arrows when viewing Upsell state

Didn't remove the arrows for the non-new date filtering.  Just toggling off `showArrows` looks bad and wasn't sure fixing it up was necessary with the upcoming changes.

## Why are these changes being made?


Latest Design:
<img width="1167" alt="Screenshot 2024-12-05 at 2 09 15 PM" src="https://github.com/user-attachments/assets/c3971b96-c49b-42b0-94cd-8ab35cbee667">

<img width="405" alt="Screenshot 2024-12-05 at 2 07 36 PM" src="https://github.com/user-attachments/assets/b6afc271-2503-49e3-a646-51f1887028b9">

Updates here:
<img width="1246" alt="Screenshot 2024-12-05 at 6 12 33 PM" src="https://github.com/user-attachments/assets/d950660f-b14e-4c94-ae05-b5ea250ae42b">

<img width="423" alt="Screenshot 2024-12-05 at 6 12 47 PM" src="https://github.com/user-attachments/assets/8fca5732-836c-49fa-a2f6-dc7fd59a1d85">


## Testing Instructions



* Open a calypso live branch and navigate to a free site's stats
* Compare the Upsell element to the Design above
* Ensure the Navigation period elements are hidden for new date filtering on upsell screen (may need to use the ?flags=stats/new-date-filtering)

## Pre-merge Checklist
